lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -1,3 +1,4 @@
+require "delegate"
```

Comment:
Do we need to explicitly [add it as a dependency in the gemspec](https://github.com/alphagov/govuk_app_config/blob/d041c5a868f092d196c126523cc1593a601da89b/govuk_app_config.gemspec#L23-L26) as well? i.e. `spec.add_dependency 'delegate'`

I imagine that in most cases this is coming in already via Rails, but being explicit means that it's guaranteed to always be available.

Although I may have misunderstood something though because https://github.com/ruby/delegate#installation suggests it needs to be in the `Gemfile`, but if it's part of the standard library, I wouldn't expect you to need to do that...

Reply:
Good point. I hit 'merge' too soon. But I think it's safe because I tested it here:  https://github.com/alphagov/bouncer/pull/273/commits/1f6f705c396e3e126e8f31076a1fa45430c6eaf3

Reply:
It's one of the default gems bundled with Ruby https://stdgems.org/ so is available by default.

I'm not sure what the expected etiquette is for gems that depend on stdgems though, it does seem prudent that they're specified as a dependency as you don't know if a later Ruby will drop them but it also feels unconventional. It's probably worth checking out some of the bigger gems for this next time we ponder about this and see if there is a precedent forming.